src/classes/STG_PanelDataImportAdvancedMapping_CTRL.cls

```diff
@@ -344,4 +358,42 @@ public with sharing class STG_PanelDataImportAdvancedMapping_CTRL extends STG_Pa
     private void addPageMessage(ApexPages.Severity severity, String message) {
         ApexPages.addMessage(new ApexPages.Message(severity, message));
     }
+
+    /*******************************************************************************************************
+    * @description Action Method to enable Gift Entry
+    * @return null
+    */
+    public PageReference enableGiftEntry() {
+        Gift_Entry_Settings__c giftEntrySettings = UTIL_CustomSettingsFacade.getGiftEntrySettings();
+        if (!giftEntrySettings.Enable_Gift_Entry__c) {
+                giftEntrySettings.Enable_Gift_Entry__c = true;
+            try{
+                upsert giftEntrySettings;
+                addPageMessage(ApexPages.Severity.CONFIRM, System.Label.geDisableGiftEntryInfo);
```

Comment:
This message reads "Gift Entry Disabled'! I think the labels need to get flipped :)